### ISSUE TITLE
DJ006 doesn't detect ModelForm.exclude on inherited forms

### ISSUE THREAD

**Issue by GitRon** [astral-sh/ruff#14147]
### Issue

I use the `DJ` extension of ruff. I just found out that this doesn't work on forms that don't use ModelForm directly but an inherited class. This relates to [DJ006](https://docs.astral.sh/ruff/rules/django-exclude-with-model-form/#django-exclude-with-model-form-dj006).

### Code example

```python
class MyModelForm(ModelForm):
    pass


class MyFancyForm(MyModelForm):
    class Meta:
        model = MyFancyModel
        exclude = ("my_field",)
```

When I change the inheritance from `MyModelForm`  to `ModelForm`, it complains as expected.

### List of keywords searched for
* DJ006

### Tried out Commands
* ruff check .\apps\billing\forms.py --isolated
* pre-commit run --all-files --hook-stage push

**Comment by MichaReiser** [comment]
It is related to https://github.com/astral-sh/ruff/issues/12865 and applies to all other Django rules. Ruff doesn't support multi-file analysis. Some rules have some hacky implementations that follow inheritance chains in the same file, but these are more one-offs and it breaks as soon as you move the classes to different files. We're working on a more refined semantic model for ruff that supports multifile analysis but it's still far out.

**Comment by GitRon** [comment]
@MichaReiser oh, thx! didn't know!

**Comment by swebra** [comment]
@MichaReiser is there an issue or discussion that can be followed to track the general release of multi-file analysis?

**Comment by MichaReiser** [comment]
There's https://github.com/astral-sh/ruff/issues/7447
